lang/wasm_alien_thread_runner.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <exception>
+#include <seastar/core/alien.hh>
+#include <seastar/core/reactor.hh>
+
+#include "lang/wasm.hh"
+#include "lang/wasm_alien_thread_runner.hh"
+
+using namespace std::chrono_literals;
+
+namespace wasm {
+
+std::optional<wasm_compile_task> task_queue::pop_front() {
+    std::unique_lock<std::mutex> lock(_mut);
```

Comment:
```suggestion
    std::unique_lock lock(_mut);
```

as the compiler is able to deduce the template parameter.

Reply:
Ok